src/nvim/api/deprecated.c

```diff
@@ -792,7 +792,8 @@ Array nvim_call_atomic(uint64_t channel_id, Array calls, Arena *arena, Error *er
 /// @param channel_id Channel id (passed automatically by the dispatcher)
 /// @param event      Event type string
 void nvim_subscribe(uint64_t channel_id, String event)
-  FUNC_API_SINCE(1) FUNC_API_REMOTE_ONLY
+// XXX: c_grammar.lua is order sensitive? FUNC_API_SINCE FUNC_API_DEPRECATED_SINCE must go first.
+  FUNC_API_SINCE(1) FUNC_API_DEPRECATED_SINCE(13) FUNC_API_REMOTE_ONLY
```

Comment:
@bfredl is it expected that `FUNC_API_SINCE FUNC_API_DEPRECATED_SINCE ` need to appear before other annotations? I didn't get very trying to make c_grammar order-insensitive.